optimizely/odp/odp_segment_api_manager.py

```diff
@@ -112,7 +112,8 @@ def __init__(self, logger: Optional[optimizely_logger.Logger] = None):
         self.logger = logger or optimizely_logger.NoOpLogger()
 
     def fetch_segments(self, api_key: str, api_host: str, user_key: str,
-                       user_value: str, segments_to_check: list[str]) -> Optional[list[str]]:
+                       user_value: str, segments_to_check: list[str],
```

Comment:
same here. we may keep the timeout constant locally instead of passing timeout for each api call.